File: android/src/main/java/com/naturalcycles/androidminimize/AndroidMinimize.java
package com.naturalcycles.androidminimize;

import android.content.Intent;
import com.getcapacitor.NativePlugin;
import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;

@NativePlugin()
public class AndroidMinimize extends Plugin {

    @PluginMethod()
    public void minimizeApp(PluginCall call) {
        Intent homeIntent = new Intent(Intent.ACTION_MAIN);
        homeIntent.addCategory( Intent.CATEGORY_HOME );
        homeIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
        getContext().startActivity(homeIntent);
        call.success();
    }
}

File: android/src/main/kotlin/com/naturalcycles/androidminimize/AndroidMinimize.kt
package com.naturalcycles.androidminimize;

import android.content.Intent
import com.getcapacitor.NativePlugin
import com.getcapacitor.PluginCall
import com.getcapacitor.PluginMethod

@NativePlugin
class AndroidMinimize : com.getcapacitor.Plugin() {
    @PluginMethod
    fun minimizeApp(call: PluginCall) {
        val homeIntent = Intent(Intent.ACTION_MAIN)
        homeIntent.addCategory(Intent.CATEGORY_HOME)
        homeIntent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP
        context.startActivity(homeIntent)
        call.success()
    }
}
